modules/caddytls/certmanagers.go

```diff
@@ -52,7 +52,7 @@ func (ts Tailscale) GetCertificate(ctx context.Context, hello *tls.ClientHelloIn
 }
 
 // canHazCertificate returns true if Tailscale reports it can get a certificate for the given ClientHello.
-func (ts Tailscale) canHazCertificate(ctx context.Context, hello *tls.ClientHelloInfo) (bool, error) {
+func (Tailscale) canHazCertificate(ctx context.Context, hello *tls.ClientHelloInfo) (bool, error) {
```

Comment:
😆  Very tempted to `git blame` who came up with that naming, fitting of Tailscale being the cool kids tho!

Reply:
That might have been me, being commissioned by Tailscale (to dev the feature, not to name it that, lol). :no_mouth: